### DISCUSSION TITLE
cannot find config in windows

### DISCUSSION THREAD

**Discussion by zhimin-z** [acheong08/ChatGPT#738] [Q&A] [answered]
I cannot run the command line since the script failed to locate the config:
![image](https://user-images.githubusercontent.com/8592144/218790874-e080a951-fe71-4111-aafc-6de1c425e7b9.png)
However, I store my local config under `$HOME/.config/revChatGPT/config.json` in my Windows 10 OS:
![image](https://user-images.githubusercontent.com/8592144/218791095-9d14bdbb-e6ca-44c7-962d-fe6deb1964b3.png)
<img width="352" alt="1676390498647" src="https://user-images.githubusercontent.com/8592144/218791314-aaee1eb5-1bc2-40fe-8e9b-9be7b819ccb9.png">
What should I do now?

**Comment by jjmpsp** [comment, marked as answer]
I looked into this issue and it seems you need to set a HOME variable within windows and assign it to your user home directory.
